client/src/components/GameEnv/GameEnv.jsx

```diff
@@ -42,8 +43,16 @@ const GameEnv = ({ setNav }) => {
     });
   }
   if ((HP <= 0 || enemyHP <= 0) && !done) {
+    if (enemyHP <= 0 && HP <= 0) {
+      axios.get(`data/wins/${user.id}/${user.total_win + 0.5}`);
```

Comment:
do these need to have .thens attached to them?

Reply:
not really

Reply:
should i?

Reply:
if it works as intended then its fine